src/components/Atoms/AddTaskButton/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+import BREAKPOINT from "../../../variables/breakpoint";
+import FONTFAMILY from "../../../variables/font_family";
+import PLUS from "../../../assets/svg/plus.svg";
+const AddTaskButton = ({ onClick }) => {
+  return (
+    <StyledAddTaskButton onClick={onClick}>
+      <Color />
+      <Task>
+        <Img src={PLUS} />
+        <Text>タスクを追加</Text>
+      </Task>
+    </StyledAddTaskButton>
+  );
+};
+export default AddTaskButton;
+
+const StyledAddTaskButton = styled.button`
+position:relative;
+  background-color: transparent;
+  border: 0;
+  cursor: pointer;
+`;
+const Color = styled.div`
+  position:absolute;
+  background-color: ${COLOR.GREEN};
+  border-radius: 100vh;
```

Comment:
```suggestion
  border-radius: 9999px;
```
vhという単位を使うと画面の高さと連動してスタイルが変化する雰囲気があるので、このように「とにかく大きい数字」であることがわかりやすいようにすると可読性が向上すると思います！
参考：https://eclair.blog/css-border-radius/